README.md

```diff
@@ -219,7 +219,7 @@ optimism_package:
         fjord_time_offset: 0
 
         # Granite fork
-        # Defaults to None - not activated - decimal value
+        # Defaults to 0 (genesis activation) - decimal value
         # Offset is in seconds
         granite_time_offset: ""
```

Comment:
if it defaults to 0 then maybe set granite_time_offset: 0 :D

Reply:
my bad, nice catch